### ISSUE TITLE
Oracle service connection

### ISSUE THREAD

**Issue by hrstoyanov** [schemaspy/schemaspy#252]
I am not sure why connecting to Oracle service needs to be so frustrating.  It has been years since Oracle abounded the SID approach ... The most straightforward way that covers __ALL ORACLE CASES__  (and perhaps all other databases) would be something like:

```
java -jar schemaSpy-6.x.x.jar -url <here goes the Oracle url. That's it!!!>
```

Why do one need  to specify users, ports, databases ??? Why do you force users to create db types, write property files and all that???

... So, I tried to figure out how to connect to Oracle service and:
- The documentation is severely lacking .
- I tried ```-t  orathin-service``` only to get something like ``` java.util.MissingResourceException: Can't find bundle for base name org/schemaspy/types/orathin-service```
- I tried  ```-t orathin``` and it again wants the damn SID!
- I tried to cheat with ``` -t orathin -db /my_oracle_con``` . Nope, does not work!

Can someone explain how to connect to Oracle service just using the command line. Better yet - document it!

**Comment by npetzall** [comment]
Hi,

You actually pointed out something with the documentation that I hadn't thought about.
We only have latest documentation, which is snapshot. The version you can download from the snapshot section in the overview/readme and not a documentation build for the rc2 release. So currently documentation is ahead of releases. Which is confusing.

I'm guessing you downloaded from the release page.
For the latest build (snapshot) use this link https://github.com/schemaspy/schemaspy/raw/gh-pages/schemaspy-6.0.0-rc2.jar

As for the requirement of properties-files, If all implementations of JDBC drivers worked identically it wouldn't be a problem (also there is some legacy code regarding parsing input that will totally destroy an url, i tried and it didn't work, we are somewhat migrating to another command-line args parser that might not have this issue).

So I would say a possible future would be -t ora -url which would be great.
There is also a PR to solve the -user -password requirement by setting -sso.
As for the database if I'm not mistaken it's used both for connecting in connectionString but it also used in the reports.

**Comment by npetzall** [comment]
@hrstoyanov have you tried the snapshot I linked.

**Comment by hrstoyanov** [comment]
I am using 6.0.0-rc2 - is this the correct one? I abused our DBAs to give me a SID for the oracle server, so I am OK now. My next issue is to figure out how to tell schemaspy not to colelct totals for each table, as this kills it. Is there option/command line switch for this? I saw nothing in the docs and ```java -jar schemaspy-6.0.0-rc2.jar -help``` tells me knothing ...

**Comment by npetzall** [comment]
Sadly it's the link in my first comment and it's name is the same as release #139 by totals you mean number of rows?

**Comment by hrstoyanov** [comment]
Yes, how can i prevent schemaspy from collecting number of rows?

**Comment by hrstoyanov** [comment]
Another question - the old schemaspy 5 was able to deduct relationships between tables just based on the column names, without the need for foreign keys. Is  schemaspy 6  capable of doing the same?

**Comment by npetzall** [comment]
It will look for implied relationships but there has been changes in how this is done resulting in issue #140 where 5.0 would pick first found PK and connect all implied relationships to that column. Changes has been done since i 6.0 that when two identical PK's are found no implied relationships are connected.

**Comment by npetzall** [comment]
I created #254 for documentation and #255 to look over and add available options to -help

**Comment by npetzall** [comment]
Was on a roll create #256 for the url on command-line.

**Comment by hrstoyanov** [comment]
you are the man !

**Comment by rafalkasa** [comment]
Hi @hrstoyanov  thank you for your contribution and share with us your feedback.
I am fully understand you. Our community is working hard to improve the SchemaSpy starting from the code ending on the documentation.

@npetzall thank you for your help and support of our community
